Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -98,6 +98,12 @@ public async Task<bool> ExecuteAsync(int retryCount, string dumpRootPath)
             }
             catch (Exception ex)
             {
+                if (ex is PageChangeAssertionException pageChangeAssertionException)
```

Comment:
The debug info coming from this exception would be helpful for every kind of exception we catch here, not just the ones happening during a page change.